README.md

```diff
@@ -0,0 +1,60 @@
+# xtm - library that implements the API for inter-thread communication
```

Comment:
1. xtm = cross-thread messaging. I suppose we should state this anywhere.
2. Key difference of this cross-thread queue from other ones — is event loop integration.

So, the title should be something like:

**XTM — a library for cross-thread communication with event loop intergation**